src/fm/cp_fm_diag.F

```diff
@@ -173,14 +175,14 @@ SUBROUTINE choose_eigv_solver(matrix, eigenvectors, eigenvalues, info)
       ! Sample peak memory
       CALL m_memory()
 
-      IF (diag_type == DIAG_TYPE_ELPA) THEN
-         CALL cp_fm_diag_elpa(matrix, eigenvectors, eigenvalues)
-      ELSE IF (diag_type == DIAG_TYPE_SCALAPACK) THEN
+      IF ((nmo < 16) .OR. (diag_type == DIAG_TYPE_SCALAPACK)) THEN
```

Comment:
Maybe add thie constant as a `PARAMETER` and put the documentation next to that constant rather than in the `\History` block to make it a bit clearer?